### ISSUE TITLE
Prototype page permalinks should allow transliteration 

### ISSUE THREAD

**Issue by stevegeek** [bridgetownrb/bridgetown#248]
As per chat:

> For permalinks in prototype generated pages, think its not possible to specify the mode for the slugify of term? (would like the slug to be in 'latin' mode to remove accented chars). Tried something like
prototype:
   
``` 
term: my_term
    collection: notes
permalink: !ruby/string:Rb |
    "/notes/#{Utils.slugify(page.data["my_term"], mode: "latin")}/"
```

> but no luck... any guidance would be appreciated! Thanks!
 
> Jared White
> I'm surprised it's not doing that already. Feel free to file an issue and I'll look into that soon.

**Bridgetown Version**:

0.19.3

**To Reproduce**

Prototype page where term contains an accented char

**Current behavior**

Accented char appears in URL

**Expected behavior**

Optionally I can opt for ASCII only URIs (as per spec tho browsers support unicode in URIs anyway)

**Comment by jaredcwhite** [comment]
This will be solvable by #243 — prototype pages will inherit a new `slugify_mode` site configuration which defaults to `pretty` but if you change it to `latin` it will fix your problem.

**Comment by jaredcwhite** [comment]
Resolved by v0.20.
